### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by anraghun**
This PR contains:

- Changes to the readme to make the documentation more readable
- Changes to the `actions_input` doc
- Modification to the action description for better search results

### CODE REVIEW

**Comment by anraghun** [comment]
@pavanvamsi3 Can you scan the changes for any typos/grammatical errors?

File: README.md [hunk 1]
```diff
@@ -1,33 +1,30 @@
-# Deploy action for Azure Kubernetes Cluster
+# Deploy action for Azure Kubernetes Service
 
-This action can be used to deploy manifests to Azure Kubernetes clusters.
+This action is dedicated to deploying workloads to Azure Kubernetes Service. A single action to take care of all the tasks required to deploy to AKS.
```
Comment (pavanvamsi3) [inline README.md:3]: 👍

File: README.md [hunk 2]
```diff
@@ -1,33 +1,30 @@
...
 
   There are two approaches for specifying the deployment target:
-
   - Kubeconfig file provided as input to the action
   - Service account approach where the secret associated with the service account is provided as input to the action
 
-  If inputs related to both these approaches are provided, kubeconfig approach related inputs are given precedence.
-- **Kubernetes create secret **(Optional)**:**  
-  This part of the action will create a [generic secret or docker-registry secret](https://kubernetes.io/docs/concepts/configuration/secret/) in Kubernetes cluster. This part is optional and only to be used when creating new secret.
+  If inputs related to both these approaches are provided, the kubeconfig approach related inputs are given precedence.
```
Comment (pavanvamsi3) [inline README.md:13]: Can this line be simplified?

File: README.md [hunk 3]
```diff
@@ -1,33 +1,30 @@
...
-
- If you are looking to automate your workflows to deploy to [Azure Web Apps](https://azure.microsoft.com/en-us/services/app-service/web/) and [Azure Web App for Containers](https://azure.microsoft.com/en-us/services/app-service/containers/), consider using [`Azure/webapps-deploy`](https://github.com/Azure/webapps-deploy) action.
+  **Note:** This action can create only one secret. For using multiple secrets in the deployment, please use the [k8s-create-secret](https://github.com/Azure/k8s-create-secret) action seperately.
+- **Deployment Configuration**  
+  This part of the action is used to deploy manifests to Azure Kubernetes Service clusters. Other configuration options such as `kubectl` version, strategy and image name overrides can also be provided. Please check the inputs for more details.
   
 ## Action inputs
-Refer to the action detail file for details about all the inputs https://github.com/Azure/deploy-to-aks/blob/releases/preview/Action_inputs.md
+For a detailed description on all the action inputs and configurations, please refer to the [action inputs doc](https://github.com/Azure/deploy-to-aks/blob/master/Action_inputs.md).
```
Comment (pavanvamsi3) [inline README.md:24]: Can we change the file name to `ACTIONS_INPUTS.md` or `actions_inputs.md`

**Pull request merged**